index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+    <head>
```

Comment:
Hey, not sure why this file is here? Its not part of the page uploaded, also not sure you need it at all!
=P